### PR TITLE
Change sunday to return correct day digit

### PR DESCRIPTION
**Pull request by amaxama**

### CODE REVIEW

File: utils/hoursUtils.js [hunk 1]
```diff
@@ -126,7 +125,9 @@ function parseTodayHours(todayOpenHours, todayCloseHours) {
 }
 
 function getTodayWeekday() {
-    return DateTime.now().toUTC().setZone(AMERICA_CHICAGO).weekday
+    const weekday = DateTime.now().toUTC().setZone(AMERICA_CHICAGO).weekday
```
Comment (amaxama) [inline utils/hoursUtils.js:128]: This is not the most elegant solution, but is a quick fix

Comment (adyates) [reply utils/hoursUtils.js:128]: Fair enough.  Although it makes more sense then the elegant-but-cryptic `weekday % 0`

**Review by adyates — APPROVED** [review]
👍

**Comment by amaxama** [comment]
Hah yeah true

**Pull request merged**